### PR TITLE
Remove barrel from sidebar

### PR DESCRIPTION
**Pull request by fonurr**

### CODE REVIEW

File: src/sidebar/barrel/tree/BarrelTreeItem.ts [hunk 1]
```diff
@@ -5,5 +5,6 @@ import Barrel from "../../../common/domain/barrel/Barrel";
 export class BarrelTreeItem extends TreeItem {
     public constructor(public barrel: Barrel) {
         super(barrel.name, vscode.TreeItemCollapsibleState.Collapsed);
+        this.contextValue = "barrel";
```
Comment (mertkahyaoglu) [inline src/sidebar/barrel/tree/BarrelTreeItem.ts:8]: Does this set `viewItem`?

**Review by mertkahyaoglu — APPROVED** [review]

File: src/sidebar/barrel/tree/BarrelTreeItem.ts (hunk 1 above)
Comment (fonurr) [reply src/sidebar/barrel/tree/BarrelTreeItem.ts:8]: Yes

File: package.json [hunk 2]
```diff
@@ -71,6 +71,12 @@
                     "when": "view == zeplin.views.barrels"
                 }
             ],
+            "view/item/context": [
+                {
+                    "command": "zeplin.sidebar.removeBarrel",
+                    "when": "view == zeplin.views.barrels && viewItem == barrel"
```
Comment (konukselim) [inline package.json:77]: Can `viewItem` have another value other than `barrel` if view is `zeplin.view.barrels`?

**Review by konukselim — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (fonurr) [reply package.json:77]: "barrels" is one of the sidebar headers.
"barrel" is one of the views in the sidebar.
In "barrels" there will also be "screens", "sections", "components" etc.

Comment (konukselim) [reply package.json:77]: Gotcha!

**Pull request merged**